core/src/main/scala/org/apache/spark/internal/io/SparkHadoopWriter.scala

```diff
@@ -60,17 +60,17 @@ object SparkHadoopWriter extends Logging {
       config: HadoopWriteConfigUtil[K, V]): Unit = {
     // Extract context and configuration from RDD.
     val sparkContext = rdd.context
-    val stageId = rdd.id
+    val rddId = rdd.id
```

Comment:
This is used as stageId in the commit protocol, IMO the original name is better.